.github/workflows/circleci.yml

```diff
@@ -251,6 +251,8 @@ jobs:
         with:
           node-version: 12.22.11
       - name: Generate devchain of previous release
+          timeout_minutes: 30
```

Comment:
shouldn't this be specifying using `nick-fields/retry@v2`

i'd expect something like this (?):
```
        uses: nick-fields/retry@v2
        with:
          timeout_minutes: 40
          max_attempts: 3
```